lib/use_cases/update_rent_charges.rb

```diff
@@ -0,0 +1,53 @@
+class UpdateRentCharges
+  def initialize(fixed_data_gateway:, rent_charges_gateway:)
+    @fixed_data_gateway = fixed_data_gateway
+    @rent_charges_gateway = rent_charges_gateway
+  end
+
+  def execute(fixed_data)
+    @fixed_data = fixed_data
+    @rc_uplift = @fixed_data[:rc_uplift].to_f
+    @year = @fixed_data[:year].to_i
```

Comment:
I suggest changing these to normal variables instead of instance variables, as this means the use case is currently stateful. Meaning if we called execute more than once, state will persist within the use case instance between them. It won't actually cause any problems now, but potentially risky if we were to add conditional logic or anything more complicated in here.